bin/generate

```diff
@@ -61,11 +57,10 @@ sed -i 's/List.listFromJson//g' lib/model/*
 
 # change some files to match requirements of pubspec.yaml specifications
 cp ../LICENSE .
-mv docs doc
-mv .analysis_options analysis_options.yaml
```

Comment:
Does generator generate right files or directory name ?
We need them to upload package on pub without issues.

Reply:
yes